persist/src/main/java/com.codeforcommunity/database/tableImpl/CommentTableDBImpl.java

```diff
@@ -130,7 +134,7 @@ public void clapComment(int postId, int commentId) {
       Connection conn = getConnection();
       // Here, we're updating the rows in the post table by the given id
       // by incrementing the clap count.
-      String sql = "UPDATE comments SET clap_count = clap_count + 1 WHERE post_id = ? AND id = ?;";
+      String sql = "INSERT INTO comment_claps (post_id, comment_id) VALUES (?, ?);";
```

Comment:
Prob dumb thing but what is point of using INSERT INTO instead of UPDATE? Is it just to showcase more functionality?

Reply:
`INSERT INTO` creates a new record in a table, but `UPDATE` changes an already existing record. Here we're doing that instead because we pulled that functionality out into a new table.